src/views/Rules/ListRules/index.jsx

```diff
@@ -780,6 +1087,27 @@ function ListRules(props) {
               ))}
             </TextField>
           </div>
+          {productChannelFilter !== ALL &&
+            searchQueries &&
+            searchQueries.length === 2 &&
+            emergencyShutoffs.find(
+              es =>
+                es.product === searchQueries[0] &&
+                es.channel === searchQueries[1]
+            ) && (
+              <EmergencyShutoffCard
+                className={classes.card}
+                emergencyShutoff={emergencyShutoffs.find(
+                  es =>
+                    es.product === searchQueries[0] &&
+                    es.channel === searchQueries[1]
+                )}
+                onEnableUpdates={handleEnableUpdates}
+                onCancelEnable={handleCancelEnableUpdates}
+                onSignoff={handleSignoffEnableUpdates}
+                onRevoke={handleRevokeEnableUpdates}
```

Comment:
Do you mind showing showing how the card looks when there are required signoffs, please? I tried adding more signoffs required in `/required-signoffs/<product>/<channel>` but that didn't work. I'm probably doing something wrong. I actually just want to check that the spacing between the yellow banner and the subtitle looks good.

Reply:
![image](https://user-images.githubusercontent.com/49649/64288705-beeb9200-cf30-11e9-8ab1-9d518e0f8cc1.png)

Reply:
👌